assets/airports/ulli.json

```diff
@@ -0,0 +1,2695 @@
+            "draw": [
+                ["OBANA*", "LI731", "LI725", "LI726", "LI727", "LI728", "LI729", "DIKOM"]
+            ]
+        },
+        "OBANA1B": {
+            "icao": "OBANA1B",
+            "name": "Obana One Bravo",
+            "entryPoints": {
+                "OBANA": [["OBANA", "A250-"]]
+            },
+            "body": [
+                ["LI744", "A70+|A140-|S225-"],
+                ["LI745", "A70"],
+                ["LI746", "A60"],
+                ["LI747", "A50+|A60-|S225-"],
+                ["LI748", "A50|S225"],
+                ["LI749", "S225"],
+                ["LI750", "S225"],
+                ["LI751", "S225"],
+                ["LI752", "A50|S225"],
+                ["OBARI", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI28L": [["BIPRI", "A25"]],
+                "ULLI28R": [["MADID", "A25"]]
+            },
+            "draw": [
+                ["OBANA*", "LI744", "LI745", "LI746", "LI747", "LI748", "LI749", "LI750", "LI751", "LI752", "OBARI"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "ulli",
+            "destination": "",
+            "category": "departure",
+            "route": "ULLI28R.KERIS1A.KERIS.R30.ABEDO",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.28,
+            "airlines": [
+                ["aul", 2.0],
+                ["ces/long", 0.71],
+                ["chh/long", 0.43],
+                ["csc/a333", 0.29],
+                ["csn/long", 0.29],
+                ["iae/su95", 0.29],
+                ["iza", 0.57],
+                ["kal/long", 0.43],
+                ["kzr/a320", 0.71],
+                ["kzr/a321", 1.0],
+                ["llm", 0.43],
+                ["llm/su95", 0.29],
+                ["pbd", 5.14],
+                ["rsy", 0.14],
+                ["sbi", 2.43],
+                ["sbi/a20n", 0.43],
+                ["sdm/a319", 11.0],
+                ["ssf/crj2", 1.0],
+                ["svr", 1.86],
+                ["uta", 1.0],
+                ["uta/b735", 0.29]
+            ]
+        },
+        {
+            "origin": "ulli",
+            "destination": "",
+            "category": "departure",
+            "route": "ULLI28R.RELNI1A.RELNI.B831.NUKSA",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.52,
+            "airlines": [
+                ["aul", 1.14],
+                ["aul/b735", 2.0],
+                ["llm", 2.0],
+                ["llm/su95", 0.29],
+                ["rlu", 0.71],
+                ["sdm/a319", 4.86],
+                ["ssf/crj2", 1.0],
+                ["syl/b738", 0.29],
+                ["tya/b738", 0.29]
+            ]
+        },
+        {
+            "origin": "ulli",
+            "destination": "",
+            "category": "departure",
+            "route": "ULLI28R.RUMOL1A.RUMOL.B924.ADOMA.B924.DILGA",
```

Comment:
This can be written as `ULLI28R.RUMOL1A.RUMOL.B924.DILGA`
No need to write intermediary waypoints, that's why airways are so useful!
Also applies to other spawnPattern routes